benchmark/Arion_Agent_Benchmark.MD

```diff
@@ -61,7 +62,36 @@ E2E programming (watch + programming) latency:
     - overall latency 5,893 ms = 5.9 seconds
     - per neighbor latency 5.89us
 
-
-![image](https://user-images.githubusercontent.com/83976250/182432696-09073c68-97fa-4829-a6b9-ad20f1763a35.png)
+![image](https://user-images.githubusercontent.com/83976250/194677460-1dbf2788-19ff-4952-8406-62743ac73373.png)
 
 As we can see, over the increasing goal-state sizes (in one version of update) the latency for each state was lowered, due to the saved/shared gRPC communication overhead across all goal-states. 
+
+
+## Arion CP performance test scenario #2 - User update neighbors to server to gateway programming E2E latency
+
+* Arion Agent Watch Latency (user simultaneously updates neighbors to server/DB)
+    - Start time – when the 1st neighbor insert/update (to server aka. ArionMaster) is called by user
+    - End time – when the Nth neighbor (last in this update) is programmed in Arion Wing ebpf, means all neighbors are programmed on gateway node
+    - Average 1 ms write latency per neighbor
```

Comment:
In general, this is a really good result. 

Could you think of creating a separate PR and check-in test codes or instruction so that other can reproduce it as well? Thanks.

Reply:
sure